CONTRIBUTING.adoc

```diff
@@ -18,7 +18,8 @@ Thank you for your contributions! Please follow this process to submit a patch:
 . Fork the repository and create a topic branch.
 . Refer to the link:https://docs.openshift.org/latest/minishift/contributing/developing.html[developer documentation] for guidelines and tips on how to build and test Minishift.
 . Submit a pull request with the proposed changes.
-  To make reviewing and integration as smooth as possible, please run the `make` targets link:https://docs.openshift.org/latest/minishift/contributing/developing.html#unit-tests[`test`], https://docs.openshift.org/latest/minishift/contributing/developing.html#format-source-code[`fmtcheck`] and https://docs.openshift.org/latest/minishift/contributing/developing.html#commit-messages[`validate_commits`] prior to submitting the pull request.
+.. The required format for the git commit message is *Issue #<issue_no> message*.
```

Comment:
@anjannath We do not need to make `Issue #<issue_no> message` bold. As it carries the same importance as other guidelines.

Reply:
My understanding was we need to differentiate it as the format for the commit message and not part of the running sentance. Making it italics/bold will give it that differance.

Reply:
@anjannath Then it would be a competition between what needs to be bold. At this point there is only the bold format but if there are several bold strings in the document in future then the overall readability will impacted. So I would suggest to keep it simple.

Reply:
@robin-owen WDYT?

Reply:
@anjannath I agree with @LalatenduMohanty here, although I would add that you should encapsulate the information in quotation marks so that it is clear what, specifically, the commit message requirements are. :)

Reply:
Thanks @robin-owen, I'll put the commit message format in quotation marks.